### DISCUSSION TITLE
change chart name

### DISCUSSION THREAD

**Discussion by gustavalentin** [netdata/netdata#13957] [Q&A] [answered]
hello.
I would need to change the chart_name of the mysql_local.slave. Netdata uses the Channel_Name of the replica, I would like to be able to use the Master_Host as chart_name

Any suggestion?

thaks!

**Comment by ilyam8** [comment, marked as answer]
Hi, @gustavalentin. No, it is not possible to change the chart name. I see the collector uses either Channel_Name (MySQL) or Connection_name (MariaDB). I think the idea was to use something unique for a connection. Different connections can have the same Master_Host. We can add Master_Host as a chart label, so you can apply different grouping using Netdata Cloud UI.

**Comment by gustavalentin** [comment]
@ilyam8 thanks for your answer. I'm going to talk to my dba team to change the channel name on the replicas servers.
